From fdd27bd3434082c47d9463a4b0b2ce9b1e1e619d Mon Sep 17 00:00:00 2001
From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Tue, 26 Jan 2016 19:20:36 +0100
Subject: [PATCH 1/2] Patch name to release -7 version

---
 debian/control.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control.em b/debian/control.em
index 175af20..f7f3c42 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -8,7 +8,7 @@ Build-Depends: debhelper (>= @(debhelper_version).0.0), @(', '.join(BuildDepends
 Homepage: @(Homepage)
 Standards-Version: 3.9.2
 
-Package: @(Package)
+Package: @(Package.replace('gazebo-','gazebo7-'))
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends))
 Description: @(Description)
-- 
2.5.0

